sparkle/tailwind.config.js

```diff
@@ -234,5 +234,11 @@ module.exports = {
   plugins: [require("@tailwindcss/forms"), require("tailwind-scrollbar-hide")],
   prefix: "s-",
   content: ["./src/**/*.{html,js,ts,jsx,tsx}"],
-  safelist: safeColorlist,
+  safelist: [
+    ...safeColorlist,
+    "s-grid-rows-2",
```

Comment:
This is necessary because Tailwind "purges" the bundles and removes all unused styles. Since we apply these styles programmatically, they are missing unless explicitly included in the safelist.

Reply:
don't we need `s-grid-cols-3` too ? or make the classname static, as cols is always 3 here

Reply:
Yep once transpiled, `s-grid-cols-3` always exist in the code.